apps/E2E/src/common/BasePage.ts

```diff
@@ -134,7 +134,17 @@ export class BasePage {
       }
       default:
       case MobilePlatform.Android:
-        // Todo
+        await browser.waitUntil(
+          async () => {
+            const buttonElementSelector = `new UiScrollable(new UiSelector().description("${TESTPAGE_BUTTONS_SCROLLVIEWER}").scrollable(true)).setMaxSearchSwipes(10).setAsVerticalList().scrollIntoView(new UiSelector().description("${this._pageButtonName}"))`;
+            const pageButton = await $(`android=${buttonElementSelector}`);
```

Comment:
If Android is going to require a different selector, I think we should modify the `By()` function at the top of this file.

I think we should create a `switch` statement such as:
`
switch(PLATFORM )
{
  case NativePlatform.Win32:
  case NativePlatform.macOS:
  case MobilePlatform.iOS:
    return await QueryWithChaining(identifier);
    break;
  case NativePlatform.Windows:
    // For some reason, the rootView node is never put into the element tree on the UWP tester. Remove this when 
        fixed.
      return await $('~' + identifier);
      break;

  case MobilePlatform.Android:
    return await QueryWithChaining(**android_specific_selector**);
`

And then, we change the signature of `QueryWithChaining(identifier)` to `QueryWithChaining(_selector_)`

and remove line 26 - `const selector = '~' + identifier;`. That won't be needed anymore since we're passing in the selector string to the method now

Reply:
Android will continue to use the same By function as is